docs/openstack/mariadb_copy.md

```diff
@@ -35,14 +35,22 @@ Define the shell variables used in the steps below. The values are
 just illustrative, use values that are correct for your environment:
 
 ```
-PODIFIED_MARIADB_IP=$(oc get -o yaml pod mariadb-openstack | grep podIP: | awk '{ print $2; }')
 MARIADB_IMAGE=quay.io/podified-antelope-centos9/openstack-mariadb:current-podified
 
-SOURCE_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
+PODIFIED_MARIADB_IP=$(oc get -o yaml pod mariadb-openstack | grep podIP: | awk '{ print $2; }')
+PODIFIED_CELL1_MARIADB_IP=$(oc get -o yaml pod ${CELL1_POD_NAME} | grep podIP: | awk '{ print $2; }')
```

Comment:
We should mention CELL1_POD_NAME prior to using it here. Or can we just use `mariadb-openstack-cell1`?

Reply:
ya thats a mistake i made in clean up

the doc version had CELL1_POD_NAME but i removed that in the test code

ill fix that to just be mariadb-openstack-cell1